packages/io/stl-deserializer/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
are these rules based on anything particular or 'stock' standard js ? (I do not work much with eslint so I confess my ignorance :)

Reply:
Let talk about this later when we clean up. Lint should be configured to run across all code, with the same configuration / rules.